docs/modules/ROOT/pages/faqs.adoc

```diff
@@ -1,83 +1,100 @@
+
+For each Cassandra write, each Cassandra replica sends one message to your Pulsar service, and executes at least one QUORUM read on your Cassandra cluster.
+
+=== What components and features are provided by Cassandra Source Connector?
+
+Cassandra Source Connector provides:
 
 * Cassandra, Elastic, Snowflake, Kinesis, Kafka, and JDBC connectors
 * A management dashboard
 * A monitoring and alerting system
 
-=== Which versions of Apache Pulsar does DataStax Cassandra Data Connector support?
+=== Which versions of Apache Pulsar does DataStax Cassandra Source Connector support?
 
-Cassandra Data Connector supports Apache Pulsar 2.7.0 and later.
+Cassandra Source Connector supports Apache Pulsar 2.7.0 and later.
 
-=== What does Cassandra Data Connector provide that I cannot get with open-source Apache Pulsar?
+=== What does the CSC provide that is not provided by Apache Pulsar?
 
-DataStax Cassandra Data Connector implements, in effect, the reverse of Apache Pulsar or DataStax CassaNdra data Connector. With CSC, the Source takes data from Cassandra and puts that date onto onto a Pulsar topic. From there, the data can be published to external platforms like Snowflake, Elastic, and so on.
+DataStax Cassandra Source Connector implements the reverse of the DataStax Cassandra Source Connector, which stores Pulsar topic updates into Cassandra.  The CSC takes data from a Cassandra table and stores that data into a Pulsar topic.  From there, the data can be published to external platforms like Snowflake, Elastic, and so on.
 
-=== Is DataStax Cassandra Data Connector an open-source project?
+=== Is DataStax Cassandra Source Connector an open-source project?
 
-Yes, DataStax Cassandra Data Connector is open source. See the <<gitHubRepos,repos FAQ>>.
+Yes, DataStax Cassandra Source Connector is open source under the Apache 2.0 License. See the <<gitHubRepos,repos FAQ>>.
 
-=== Which Kubernetes platforms are supported by DataStax Cassandra Data Connector?
+=== Which Kubernetes platforms are supported by the CSC?
 
-They include Minikube, K8d, Kind, Google Kubernetes Engine (GKE), Microsoft Azure Kubernetes Service, Amazon Kubernetes Service (AKS), and other commonly used platforms.
+The CSC supports most commonly used Kubernetes platforms, including:
+
+* Minikube
+* K8d
+* Kind
+* Google Kubernetes Engine (GKE)
+* Microsoft Azure Kubernetes Service
+* Amazon Kubernetes Service (AKS), and other commonly used platforms.
 
 [#gitHubRepos]
-=== Where is the DataStax Cassandra Data Connector public GitHub repo?
+=== Where is the CSC public GitHub repo?
 
-The source for this FAQs document is co-located with the Cassandra Data Connector repo code, examples, and additional documentation: access the [repo here](https://github.com/riptano/cdcrep).
+The source for this FAQ and related CSC documents are co-located with the Cassandra Source Connector repo code, examples, and additional documentation: access the [repo here](https://github.com/riptano/cdcrep).
```

Comment:
Is this accurate? `riptano` repos aren't typically public. I think it should be linked as shown below instead.

```suggestion
The link:https://github.com/datastax/cassandra-source-connector/[Cassandra Source Connector repo] includes code, examples, and additional documentation.
```

Reply:
You're correct - that was an old link.